### PR TITLE
Add guide for setting custom res with nvidia on linux

### PR DESCRIPTION
**Pull request by ehabajmal**
## Description
Added guide for setting custom resolution with nvidia on Linux. The Linux xrandr guide did not work for me with an nvidia gpu, only by using the script and commands I provided was I able to set custom resolutions on app startup/shutdown.

### Screenshot
N/A

### Issues Fixed or Closed
N/A

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I want maintainers to keep my branch updated

### CODE REVIEW

File: docs/source/about/guides/app_examples.rst [hunk 1]
```diff
@@ -197,6 +197,25 @@ Changing Resolution and Refresh Rate (Linux - KDE Plasma - Wayland and X11)
...
++----------------------+----------------------------------------------------------------------------------------------------------------------------------+
+
+The ``set-custom-res.sh`` will have this content:
+   .. code-block:: bash
+
+      #!/bin/bash
+
+      # Change HDMI-1 to whatever your monitor is
+      nvidia-settings -a CurrentMetaMode="HDMI-1: nvidia-auto-select { ViewPortIn=$1, ViewPortOut=$1+0+0 }"
+
```
Comment (ReenigneArcher) [inline docs/source/about/guides/app_examples.rst:218]:
```suggestion
Changing Resolution (Linux - NVIDIA)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

+----------------------+------------------------------------------------------------------------------------------------------+
| **Field**            | **Value**                                                                                            |
+----------------------+------------------------------------------------------------------------------------------------------+
| Command Preparations | Do: ``sh -c "${HOME}/scripts/set-custom-res.sh ${SUNSHINE_CLIENT_WIDTH} ${SUNSHINE_CLIENT_HEIGHT}"`` |
|                      +------------------------------------------------------------------------------------------------------+
|                      | Undo: ``sh -c "${HOME}/scripts/set-custom-res.sh 3840 2160"``                                        |
+----------------------+------------------------------------------------------------------------------------------------------+

The ``set-custom-res.sh`` will have this content:
   .. code-block:: bash

      #!/bin/bash

      # Get params and set any defaults
      width=${1:-1920}
      height=${2:-1080}
      hdmi=${3:-HDMI-1}
      nvidia-settings -a CurrentMetaMode="${hdmi}: nvidia-auto-select { ViewPortIn=${width}x${height}, ViewPortOut=${width}x${height}+0+0 }"
```

**Review by ReenigneArcher — CHANGES_REQUESTED** [review]

File: docs/source/about/guides/app_examples.rst [hunk 2]
```diff
@@ -197,6 +197,27 @@ Changing Resolution and Refresh Rate (Linux - KDE Plasma - Wayland and X11)
...
+The ``set-custom-res.sh`` will have this content:
+   .. code-block:: bash
+
+      #!/bin/bash
+
+      # Get params and set any defaults
+      width=${1:-1920}
+      height=${2:-1080}
+      hdmi=${3:-HDMI-1}
+      nvidia-settings -a CurrentMetaMode="${hdmi}: nvidia-auto-select { ViewPortIn=${width}x${height}, ViewPortOut=${width}x${height}+0+0 }"
```
Comment (ReenigneArcher) [inline docs/source/about/guides/app_examples.rst:220]:
```suggestion
      nvidia-settings -a CurrentMetaMode="${hdmi}: nvidia-auto-select { ViewPortIn=${width}x${height}, ViewPortOut=${width}x${height}+0+0 }"

```

Comment (ehabajmal) [reply docs/source/about/guides/app_examples.rst]: Added newline and changed name of var "hdmi" to "output" for clarity

**Review by ReenigneArcher — APPROVED** [review]

**Comment by ReenigneArcher** [comment]
Why was this closed?

**Comment by ReenigneArcher** [comment]
You don't have permission to merge

**Pull request merged**